PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,48 @@
+# Pull Request Template
+
+Please use this Template for all Pull Requests to this repository.
+
+```markdown
+Reason for PullRequest
```

Comment:
Same as above, duplicated, mind, this file is directly copied to the comment section upon creating a new PR.